templates/d.erb

```diff
@@ -9,6 +9,11 @@ PATH=/usr/local/bin:/usr/local/sbin:/nail/sys/bin:/usr/sbin:/usr/bin:/sbin:/bin
 <% else -%>
 PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/local/sbin
 <% end -%>
+<% if ! @env.empty? -%>
+<% @env.each do |variable, value| %>
```

Comment:
btw this will also fix tests which are failing because ruby 1.8 doesn't support `.each` over hashmap